src/algebra/ordered_group.lean

```diff
@@ -1262,17 +1455,27 @@ sub_lt_iff_lt_add'.trans (lt_add_iff_pos_left _)
 
 end ordered_add_comm_group
 
-/--
-The `add_lt_add_left` field of `ordered_add_comm_group` is redundant, but it is in core so
-we can't remove it for now. This alternative constructor is the best we can do.
+/-
+TODO:
+The `add_lt_add_left` field of `ordered_add_comm_group` is redundant,
+and it is no longer in core so we can remove it now.
+This alternative constructor is the best we could do.
```

Comment:
Surely ... "is a workaround until someone fixes this"?

Reply:
I guess change "best we *can* do" to "best we *could* do" is not very strong wording (-;